lldb/source/Plugins/Architecture/AArch64/ArchitectureAArch64.cpp

```diff
@@ -108,10 +104,11 @@ bool ArchitectureAArch64::ReconfigureRegisterInfo(DynamicRegisterInfo &reg_info,
   if (!vg_reg_value && !svg_reg_value)
     return false;
 
+  auto regs = reg_info.registers<DynamicRegisterInfo::reg_collection_range>();
```

Comment:
nit: If you specify the type of `regs` (instead of using `auto`), you shouldn't have to specify the template argument, and just call `reg_info.registers()`.

Reply:
I couldn't figure out how to get that to work:
```
/home/david.spickett/llvm-project/lldb/source/Plugins/Architecture/AArch64/ArchitectureAArch64.cpp:107:61: error: no matching member function for call to 'registers'
  DynamicRegisterInfo::reg_collection_range regs = reg_info.registers();
                                                   ~~~~~~~~~^~~~~~~~~
/home/david.spickett/llvm-project/lldb/include/lldb/Target/DynamicRegisterInfo.h:93:27: note: candidate template ignored: couldn't infer template argument 'T'
  template <typename T> T registers() = delete;
                          ^
1 error generated.
```
Feel free to do it yourself if you know what the syntax should be.